src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs

```diff
@@ -4,21 +4,42 @@
 using Microsoft.Win32.SafeHandles;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
-using System.Diagnostics.Contracts;
 using System.Runtime.InteropServices;
 using System.Security;
-using System.Threading;
-using System.Threading.Tasks;
 
 namespace System.IO.Pipes
 {
     public abstract partial class PipeStream : Stream
     {
+        private const string PipeDirectoryPath = "/tmp/";
+
+        internal static string GetPipePath(string serverName, string pipeName)
```

Comment:
Is there any sort of validation performed on this argument by this method's caller?  It would be good to ensure that this is literally a name and not a path.